TestAssemblies/SimpleTest/AsyncClass.cs

```diff
@@ -5,17 +5,17 @@ namespace SimpleTest {
     public class AsyncClass {
         [Interceptor]
         public async Task SimpleAsyncMethod() {
-            await Task.Delay(1);
+            await Task.Delay(1).ConfigureAwait(false);
```

Comment:
Without this I wasn't able to run the tests using the xUnit visual studio runner.